src/core/tsi/test_creds/leaf_signed_by_intermediate.cnf

```diff
@@ -0,0 +1,12 @@
+[req]
+distinguished_name = req_distinguished_name
+req_extensions = v3_req
+prompt = no
+
+[req_distinguished_name]
+CN = *.test.google.com.au
+
+[v3_req]
+keyUsage = critical, digitalSignature, keyEncipherment
+extendedKeyUsage = clientAuth, serverAuth
+basicConstraints = critical, CA:false
```

Comment:
Same here RE blank line to make the linter happy. :)

Reply:
Done